paddle/fluid/operators/box_coder_op.cc

```diff
@@ -87,8 +96,10 @@ class BoxCoderOpMaker : public framework::OpProtoAndCheckerMaker {
         "OutputBox",
         "(LoDTensor or Tensor) "
         "(Tensor) The output of box_coder_op, a tensor with shape [N, M, 4] "
-        "representing the result of N target boxes encoded/decoded with "
-        "M Prior boxes and variances.");
+        "representing the result of N target boxes encoded with "
+        "M Prior boxes and variances when code_type is 'encode_center_size'. "
+        "N represents the batchsize and M represents the deocded boxes when "
```

Comment:
batchsize -> batch size
M represents the deocded boxes -> M represents the number of decoded boxes of each instance.

Reply:
done